_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-cayman
```

Comment:
This theme is fine, but would like to keep the black/light-grey/white aesthetics consistent with the [logo](https://camo.githubusercontent.com/15417a4f11329c994eee9f58b842be48d22dc0ee/68747470733a2f2f63646e2d696d616765732d312e6d656469756d2e636f6d2f6d61782f313630302f312a444b533570596673417a2d4834356d79766e575756772e676966). Could we do that customization in this PR?

Reply:
The customizations can be done but need to change the logo file for it, we can easily do that in upcoming PR with other changes that we have discussed